doc/getting_started.md

```diff
@@ -0,0 +1,25 @@
+# Getting started with PacSim
+
+To use PacSim, you need to create a message converter node to match your own interfaces with the simulator. You will need to advertise and publish to some topics. Note that some topics are not directly used in the simulator.
+
+| TOPIC                        | MESSAGE TYPE               | DESCRIPTION                                                                           | USED?  |
+|------------------------------|----------------------------|---------------------------------------------------------------------------------------|---|
+| /pacsim/steering_setpoint    | pacsim::msg::StampedScalar | Target steering angle at the steering wheel/sensor (rad)                              | YES |
+| /pacsim/torques_max          | pacsim::msg::Wheels        | Upper bound torque value at each wheel's motor (Nm)                                   | YES |
+| /pacsim/powerground_setpoint | pacsim::msg::StampedScalar | Powered ground multiplier. 0 for no powered aero, 1 for full capacity (Dimensionless) | YES |
+| /pacsim/torques_min          | pacsim::msg::Wheels        | Lower bound torque value at each wheel's motor (Nm)                                   | NO |
+| /pacsim/wheelspeed_setpoints | pacsim::msg::Wheels        | Target wheel speeds at each wheel's motor (RPM)                                       | NO |
+
+This simulator does not model the interface of a motor controller. It is highly recommended you implement your own internal control loop that controls the amount of torque applied to each wheel to reach a given target wheel speed. Moreover, `torques_max` can be seen more as the "current torque request sent", rather than the upper bound torque value at each wheel's motor.
```

Comment:
I would use a more generalized formulation for this, kind of like that:

The interfaces are provided such that motor controllers with an internal control loop for the wheelspeeds (e.g. AMK or Electrophorus Engineering) can be fully modelled in the simulator.

The default model doesn't contain this control loop. The values provided for `torques_max` are the actual torques applied to the wheels.